genon/preprocessor/facade/convert_processor.py

```
@@ -1526,6 +1526,133 @@ def chunk(self, dl_doc: DoclingDocument, **kwargs: Any) -> Iterator[BaseChunk]:
+
+def _gr_doc_text(document: DoclingDocument) -> str:
+    """분류 워크플로우로 보낼 문서 전체 텍스트. 그림만 제외. 표는 마크다운으로 포함한다
+    (표 셀 PII 도 분류/마스킹 대상 — 청크도 표를 마크다운으로 담으므로 quote 매칭 가능)."""
+    parts = []
+    for it, _ in document.iterate_items():
+        if isinstance(it, PictureItem):
+            continue
+        if isinstance(it, TableItem):
+            try:
+                t = it.export_to_markdown(document)
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)